### ISSUE TITLE
Login Link Email: Fix the "Explore the best content on WordPress" heading styling on mobile

### ISSUE THREAD

**Issue by ivan-ottinger** [Automattic/wp-calypso#74255]
The "Explore the best content on WordPress" heading is smaller on mobile than it should be:

| Design | Template |
| ------------- | ------------- |
| ![Image](https://user-images.githubusercontent.com/25105483/223979502-17a7ed44-6eb6-4b12-b8c4-0ede014e6811.png) | ![Image](https://user-images.githubusercontent.com/25105483/223979501-b4853ea9-8b27-4857-837f-df1753e973d6.png) |

The correct typography (from the design) is:

```
font-family: SF Pro Display;
font-size: 28px;
font-weight: 700;
line-height: 34px;
letter-spacing: 0px;
text-align: left;
```

Link to the design: inDLaEQV8jJ21O4WXawIsJ-fi-712-13475

**Comment by JuanLucha** [comment]
Fixes here: D135735-code

**Comment by ivan-ottinger** [comment]
Marking as ✅ done. Nice job, Juan! 🙂
